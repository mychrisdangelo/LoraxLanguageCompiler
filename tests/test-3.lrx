int increment(int b)
{
	return b + 1;
}


int main()
{
	int a;
	a = 1;
	a = increment(a);
	return a;
}