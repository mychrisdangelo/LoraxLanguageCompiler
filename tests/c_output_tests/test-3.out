#include "lrxlib.h"
int main();
int increment(int);

int main()
{
int a_2 = 0;
int __tmp_int_5 = 0;
int __tmp_int_4 = 0;
int __tmp_int_3 = 0;

__tmp_int_5 = 1;

a_2 = __tmp_int_5;

__tmp_int_4 = increment( a_2 );

a_2 = __tmp_int_4;

return a_2;
return __tmp_int_3;

}
int increment(int b_1)
{
int __tmp_int_2 = 0;
int __tmp_int_1 = 0;
int __tmp_int_0 = 0;

__tmp_int_1 = 1;

__tmp_int_2 = b_1 + __tmp_int_1;

return __tmp_int_2;
return __tmp_int_0;

}
